flatpak/me.brokenby.PdfMetadataEditor.Batch.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Batch Pdf Metadata Editor
+Comment=Simple Editor for PDF metadata (Batch Mode)
+Exec="/app/bin/Batch Pdf Metadata Editor"
```

Comment:
Don't use absolute path and preferably name the executable something that doesn't have spaces

Reply:
Removed the absolute path.

Unfortunately due to a limitation of jpackage ( the tool used to build the program) I can't make the executable name different than the general name,  and this results in executable with spaces.

Once jpackage is fixed, I for sure will change the executable name,